[2023-10-22T06:30:00.172+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=nhl_stats_save/run_id=manual__2023-10-22T06:29:57.666509+00:00/task_id=save_nhl_stats permission to 509
[2023-10-22T06:30:00.193+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=nhl_stats_save/run_id=manual__2023-10-22T06:29:57.666509+00:00/task_id=save_nhl_stats permission to 509
[2023-10-22T06:30:00.216+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nhl_stats_save.save_nhl_stats manual__2023-10-22T06:29:57.666509+00:00 [queued]>
[2023-10-22T06:30:00.223+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nhl_stats_save.save_nhl_stats manual__2023-10-22T06:29:57.666509+00:00 [queued]>
[2023-10-22T06:30:00.223+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-22T06:30:00.234+0000] {taskinstance.py:1382} INFO - Executing <Task(NhlStatsSaveOperator): save_nhl_stats> on 2023-10-22 06:29:57.666509+00:00
[2023-10-22T06:30:00.239+0000] {standard_task_runner.py:57} INFO - Started process 1618 to run task
[2023-10-22T06:30:00.242+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'nhl_stats_save', 'save_nhl_stats', 'manual__2023-10-22T06:29:57.666509+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/nhl_stats_save.py', '--cfg-path', '/tmp/tmp64c1kvhw']
[2023-10-22T06:30:00.244+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask save_nhl_stats
[2023-10-22T06:30:00.254+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:193: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-10-22T06:30:00.276+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=nhl_stats_save/run_id=manual__2023-10-22T06:29:57.666509+00:00/task_id=save_nhl_stats permission to 509
[2023-10-22T06:30:00.278+0000] {task_command.py:416} INFO - Running <TaskInstance: nhl_stats_save.save_nhl_stats manual__2023-10-22T06:29:57.666509+00:00 [running]> on host 06f8d34233ae
[2023-10-22T06:30:00.334+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='babintseva' AIRFLOW_CTX_DAG_ID='nhl_stats_save' AIRFLOW_CTX_TASK_ID='save_nhl_stats' AIRFLOW_CTX_EXECUTION_DATE='2023-10-22T06:29:57.666509+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-22T06:29:57.666509+00:00'
[2023-10-22T06:30:00.796+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/opt/airflow/plugins/nhl_stats_save_operator.py", line 101, in execute
    self.save_data(data)
  File "/opt/airflow/plugins/nhl_stats_save_operator.py", line 74, in save_data
    (team, stats, splits) = self.parse_data(df)
  File "/opt/airflow/plugins/nhl_stats_save_operator.py", line 64, in parse_data
    'shooting_pctg': jsSplit.get('stat')['shootingPctg'],
KeyError: 'shootingPctg'
[2023-10-22T06:30:00.806+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=nhl_stats_save, task_id=save_nhl_stats, execution_date=20231022T062957, start_date=20231022T063000, end_date=20231022T063000
[2023-10-22T06:30:00.818+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 7 for task save_nhl_stats ('shootingPctg'; 1618)
[2023-10-22T06:30:00.855+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-22T06:30:00.875+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
